iModelCore/ecobjects/src/ECSchema.cpp

```diff
@@ -834,7 +834,7 @@ ECObjectsStatus ECSchema::CreateEntityClass (ECEntityClassP& pClass, Utf8StringC
 //---------------------------------------------------------------------------------------
 // @bsimethod
 //---------------+---------------+---------------+---------------+---------------+-------
-ECObjectsStatus ECSchema::CreateMixinClass (ECEntityClassP& pClass, Utf8StringCR name, ECEntityClassCR appliesTo)
+ECObjectsStatus ECSchema::CreateMixinClass (ECEntityClassP& pClass, Utf8StringCR name, ECEntityClassCR appliesTo, ECSchemaReadContextR schemaContext)

```

Comment:
This is one spot that could potentially come back and bite us. I believe the method is commonly used in many spots, I don't feel good about changing its signature.
As an alternative, the method could require the schema to have a reference to CoreCustomAttributes and fail if the reference is missing. We could then add this overload alongside the method. So we don't break anybody. However, this could lead to runtime errors which is even worse :-) So I guess I'm fine but not happy with this...